http/vibe/http/websockets.d

```diff
@@ -699,12 +699,14 @@ final class WebSocket {
 			reason = reason[0 .. min($, 123)];
 
 		if (connected) {
-			send((scope msg) {
-				m_sentCloseFrame = true;
-				if (code != 0)
-					msg.write(std.bitmanip.nativeToBigEndian(code));
-					msg.write(cast(const ubyte[])reason);
-				}, FrameOpcode.close);
+			try {
+				send((scope msg) {
+					m_sentCloseFrame = true;
+					if (code != 0)
+						msg.write(std.bitmanip.nativeToBigEndian(code));
+						msg.write(cast(const ubyte[])reason);
+					}, FrameOpcode.close);
+			} catch (Exception) {}
```

Comment:
Would be good to add a `logDiagnostic("Failed to send active web socket close frame: %s", e.msg);` here.

Reply:
I'd consider this normal behaviour, but I added the log message, sure.
Also, I added a commit where I fix the close reply (the indentation has been misleading…).